tests/next_tests/integration_tests/multi_feature_tests/iterator_tests/test_fvm_nabla.py

```diff
@@ -396,17 +370,11 @@ def nabla_sign(n_nodes, out_MXX, out_MYY, pp, S_MXX, S_MYY, vol, node_index, is_
 
 
 @pytest.mark.requires_atlas
```

Comment:
@egparedes Can you please explain why we need the marker `requires_atlas` only on this test case? Should we add the `uses_atlas_tables` marker to any other backend, in addition to those 4 in baseline code?

Reply:
I think it is just an oversight. In principle, all tests in the file should need the `requires_atlas` mark to skip them if atlas is not installed but the `importorskip(atlas4py)` statement at the beginning of the file does basically the same automatically, and that's why the tests in this file didn't failed. `importorskip` is useful if we really want to skip ALL the tests from a file if a dependency is missing, but the custom markers give us finer grain control.

BTW, In this case it looks like `uses_atlas_tables` and `requires_atlas` are currently equivalent, right? Maybe we should delete one of them.

Reply:
Good catch, by the way! :facepalm: